reya-rust-sdk/src/data_types.rs

```diff
@@ -16,6 +73,32 @@ pub enum CommandType {
     TransferMarginAccount,
 }
 
+#[allow(dead_code)]
+#[repr(u8)]
+#[derive(PartialEq, Eq, Clone, Copy, Debug)]
+pub enum OrderType {
+    StopLoss = 0,
+    TakeProfit,
```

Comment:
let's delete TakeProfit since we don't have that on-chain in the enum to keep consistent